purecharity-wp-trips/purecharity-wp-trips.php

```diff
@@ -129,4 +129,32 @@ function gt_force_template() {
   $options = get_option( 'purecharity_trips_settings' );
   include(TEMPLATEPATH . '/' . $options['single_view_template']);
   exit;
+}
+
+/*
+ * Plugin updater using GitHub
+ *
+ * Auto Updates through GitHub
+ *
+ * @since   1.0.4
+ */
+add_action( 'init', 'purecharity_wp_trips_updater' );
+function purecharity_wp_trips_updater() {
+  if ( is_admin() ) { 
+    $tr_config = array(
+      'slug' => plugin_basename( __FILE__ ),
+      'proper_folder_name' => 'purecharity-wp-trips',
+      'api_url' => 'https://api.github.com/repos/purecharity/pure-trips',
+      'raw_url' => 'https://raw.github.com/purecharity/pure-trips/master',
+      'raw_url' => 'https://raw.githubusercontent.com/purecharity/pure-trips/master/purecharity-wp-trips/',
```

Comment:
I'm not sure if this is intentional but there are two entries for `raw_url` here